### ISSUE TITLE
Re-open apps after policy finishes

### ISSUE THREAD

**Issue by cubandave** [cubandave/Jamf-Interaction-Toolkit#3]
If the the only actions were to quit or block application 
After all the actions finish then re open the applications the user was forced to quit

**Comment by cubandave** [comment]
need to decided on dynamics for the are you sure message and check the type of kill to see if it can quit safely to ask the user to save or not
also need to see about decigind which app to relaunch if there are multiple there. for example "Adobe Illustrator.app"

**Comment by cubandave** [comment]
add inteligence to quit the apps before the are you sure message and then change the are you sure message to be just save you work and continue

also add to the incativtify loop to break and write a if the user quits the apps then re open it by makking the the app2relaunch equal the curernt list

**Comment by cubandave** [comment]
Work in the mechanism whinnblick is active instead

**Comment by cubandave** [comment]
add an activate then quit so the save dialogs appear at the front of the user

**Comment by cubandave** [comment]
de9f44f135c23ed6260ac3e47cca7c623da44068 Safe quit and reopen in background functions added
39d54daebbe0c970fc7c399bbcc56420afeed83b detection for if the app is quit

**Comment by cubandave** [comment]
update the dialog section
```

if [[ "${apps2quit[@]}" == *".app"*  ]] && [[ "$checks" != *"custom"* ]] ; then
	PostponeMsg+="• Please quit:
$apps4dialogquit

"
fi

if [[ "${apps2ReOpen[@]}" == *".app"*  ]] && [[ "$checks" != *"custom"* ]] ; then
	PostponeMsg+="• These apps will reopen after the $action:
$apps4dialogreopen

"
fi
```

**Comment by cubandave** [comment]
working but restart detections is now not working

**Comment by cubandave** [comment]
fixed the wording in cf48dee9ca7e7254030e6b2d8449589be6ac69ef
